src/views/partials/doc-resource-traits.ejs

```diff
@@ -59,8 +59,8 @@
     </tr>
     <tr>
       <td align="left">desc</td>
-      <td align="left">The description of the trait.</td>
-      <td align="left">array</td>
+      <td align="left">A brief description of the trait.</td>
```

Comment:
It's the full description from the SRD. So `brief` doesn't feel like it's accurate?

Reply:
For example, this is the Breath Weapon trait:
```
    "desc": [
      "You can use your action to exhale destructive energy. Your draconic ancestry determines the size, shape, and damage type of the exhalation.",
      "When you use your breath weapon, each creature in the area of the exhalation must make a saving throw, the type of which is determined by your draconic ancestry. The DC for this saving throw equals 8 + your Constitution modifier + your proficiency bonus. A creature takes 2d6 damage on a failed save, and half as much damage on a successful one. The damage increases to 2d6 at 6th level, 3d6 at 11th level, and 5d6 at 16th level.",
      "After you use your breath weapon, you cannot use it again until you complete a short or long rest."
    ],
```

Reply:
Makes sense, will update.

Feels a little strange that the `desc` field is sometimes an actual array of multiple strings, but other times an always one element array of strings.

Reply:
I feel that, but that's mainly to support multiple paragraphs.

Reply:
Totally makes sense now, I wasn't thinking of it in terms of paragraphs before. Thanks!

Reply:
On this note, it would be nice at some point to migrate to using markdown for long-form text fields. Especially seeing as some `desc`s contain tables and the like.

Reply:
Oh yes!

Reply:
Magic Items follows this approach but I'm sure the others could use some love.